src/main/java/edu/ie3/datamodel/utils/validation/ValidationUtils.java

```diff
@@ -54,18 +59,39 @@ protected static NotImplementedException checkNotImplementedException(Object obj
    * fulfill the checking task, based on the class of the given object.
    *
    * @param obj Object to check
-   * @throws edu.ie3.datamodel.exceptions.NotImplementedException if an unknown class is handed in
+   * @return a list of try objects either containing a {@link ValidationException} or an empty
+   *     Success
    */
-  public static void check(Object obj) {
-    checkNonNull(obj, "an object");
-    if (AssetInput.class.isAssignableFrom(obj.getClass())) checkAsset((AssetInput) obj);
-    else if (GridContainer.class.isAssignableFrom(obj.getClass()))
-      GridContainerValidationUtils.check((GridContainer) obj);
-    else if (GraphicInput.class.isAssignableFrom(obj.getClass()))
-      GraphicValidationUtils.check((GraphicInput) obj);
-    else if (AssetTypeInput.class.isAssignableFrom(obj.getClass()))
-      checkAssetType((AssetTypeInput) obj);
-    else throw checkNotImplementedException(obj);
+  public static Try<Void, ? extends ValidationException> check(Object obj) {
```

Comment:
It'd be great if, like with inputs, we don't force the `Try` class on users of this validation functionality. So I'd rather prefer a method signature like this:
```java
public static void check(Object obj) throws ValidationException
```
This is also important for all the tests using `noExceptionThrown()` to work.

On that note, `ValidationException` should really _not_ be a `RuntimeException`, so that `throws ...` has to be declared wherever we actually throw.